ui/tests/e2e/fixtures/environmentsPage.js

```diff
@@ -0,0 +1,204 @@
+import { expect } from '@playwright/test';
+import { ENV } from '../env';
+
+export class EnvironmentPage {
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+  constructor(page) {
+    this.page = page;
+  }
+  async navigate() {
```

Comment:
Let's do navigation to page directly on test level, because it's make easier on debuging